.github/workflows/ci.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         pip install -e .
         python -m playwright install
     - name: Lint
-      run: pre-commit run --all-files
```

Comment:
I see no reason to use a third-party action when the current implementation will do the same.

Reply:
Ok but the pre-commit action caches the actions and speeds up it from 40 sec to 8 sec. Anyway if you want to remove it I will

Reply:
ok lets keep it!